app/scripts/metamask-controller.js

```diff
@@ -2064,6 +2064,23 @@ export default class MetamaskController extends EventEmitter {
         blockExplorerUrl,
       },
     );
+
+    this.metaMetricsController.trackEvent({
+      event: 'Custom Network Added',
+      category: EVENT.CATEGORIES.NETWORK,
+      referrer: {
+        url: rpcUrl,
+      },
+      sensitiveProperties: {
+        chain_id: chainId,
+        rpc_url: rpcUrl,
+        network_name: chainName,
+        network: rpcUrl,
+        symbol: ticker,
+        block_explorer_url: blockExplorerUrl,
+        source: EVENT.SOURCE.NETWORK.POPULAR_NETWORK_LIST,
```

Comment:
@brad-decker this just follows the implementation in `app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js`

Should that be changed as well?

Reply:
Yes there is no reason for these to be sensitiveProperties at all.... I mean we can maybe do some sleuth work about which chains a user holds assets in and which ones are in their metamask but I feel this isn't providing much value as sensitiveProperties.

Reply:
Okay, makes sense. I still think there is a case to be made to have rpc_url in sensitive properties (see this thread for some related thinking https://consensys.slack.com/archives/G01FXS64DFA/p1657543140223419)

If our data was compromised, it would be better if user metrics ids were not linked to rpc_url, because the rpc endpoint is a possible attack vector. So, for example, if I was trying to hack a specific person and knowing what rpc endpoint they use opens up at least two attacked vectors: compromising that endpoint or phishing them into adding a really similar but different looking endpoint.

On the flipside, I don't think the rpc_url tells us anything interesting in relation to other non-sensitive data that the chainId doesn't tell us. We are only interested in rpc_urls for questions that are answered by there aggregate.

So I suggest moving all properties out of sensitive properties except that one.

@filipsekulic can you also change the `sendMetrics` call in `app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js` in this way?

Reply:
Fixed. You can now see the changes. 
Hope I understood you clearly and made appropriate changes.